.travis.yml

```diff
@@ -0,0 +1,29 @@
+sudo: required
```

Comment:
This is not too much important, but we could make this work without sudo by replacing `sudo ln -s ...` with `export PATH=$PATH:/tmp/nextflow`